package.json

```diff
@@ -133,21 +130,21 @@
     "@types/http-errors": "^2.0.4",
     "@types/jest": "^29.5.12",
     "@types/jsonwebtoken": "^9.0.6",
-    "@types/node": "^22.5.1",
+    "@types/node": "^20.16.5",
```

Comment:
is this intential? downgrading or opening up for lower versions? "@types/node": "^20.16.5",